src/resolver/pathsLookup.ts

```diff
@@ -82,6 +84,11 @@ function getIndexFiles(props: IPathsLookupProps): DirectoryListing | undefined {
 }
 
 export function pathsLookup(props: IPathsLookupProps): ILookupResult {
+  const fileDir =
```

Comment:
Are you sure that we need that? Maybe a fix to the regex will do?

Reply:
IDK, I added a test case based on the structure of `sha.js` to show my motivation for this change. I encounter `ISDIR: illegal operation on a directory for sha.js` when I try to include it in my app bundle.
I didn't find a better way, but everything that will pass the new test case will make me happy.